### PR TITLE
Created Anagram code in java

### PR DESCRIPTION
**Pull request by RuchiGitHu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
  Yes
### Description

(Write your answer here.)

### Checklist

- [✓] I've read the contribution guidelines.
- [✓] I've checked the issue list before deciding what to submit.
- [✓] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
Contributing for issue #1969 .

### CODE REVIEW

File: Java/cp/Anagram_Java.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package HACKER_RANK;
```
Comment (atarax665) [inline Java/cp/Anagram_Java.java:1]: Remove this and add a beginner description of the problem. You can also add the link of the problem if possible.

File: Java/cp/Anagram_Java.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		Scanner scan = new Scanner(System.in);
+		System.out.println("Please enter the first String : ");
+		String a = scan.next();
+		System.out.println("Please enter the Second String : ");
+		String b = scan.next();
+		scan.close();
+		boolean ret = isAnagram(a, b);
+		System.out.println((ret) ? "String are Anagrams" : "Strings are Not Anagrams");
+	}
+}
```
Comment (atarax665) [inline Java/cp/Anagram_Java.java:43]: Add sample input/output and time, space complexity of your solution. Please refer to any recently merged java PR to get a better idea of the java template that we follow here.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Update README.md too.

File: Java/cp/Anagram_Java.java (hunk 2 above)
Comment (RuchiGitHu) [reply Java/cp/Anagram_Java.java:43]: Ok @atarax665 ,I am updating as per your suggestion.

**Comment by RuchiGitHu** [comment]
> Update README.md too.

Please let me know which readme.md file you want to update( if possible, attach a link of readme.md file). What type of updates you want to do so? 
I will be grateful to you if you tell me something more about the required changes for readme.md file...
Thanks in Advance...

**Review by atarax665 — CHANGES_REQUESTED** [review]
Please update [this](https://github.com/RuchiGitHu/NeoAlgo/blob/anagram_java/Java/README.md) README. Add hyperlink of your contributed program under Competitive Programming. Please refer to any recently merged PR to get an idea.

**Comment by atarax665** [comment]
#2217

**Pull request closed without merging**